pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -279,6 +283,7 @@ func Run() {
 		log.Fatal(err)
 	}
 	subAppNetworkConfig.ModifyHandler = handleAppNetworkConfigModify
+	subAppNetworkConfig.CreateHandler = handleAppNetworkCreate
```

Comment:
Why not name the functions consistently i.e., handleAppNetworkConfigCreate in this case?